hazelcast/serialization/service.py

```diff
@@ -444,7 +444,8 @@ def lookup_custom_serializer(self, obj_type):
         serializer = self._type_dict.get(obj_type, None)
         if serializer is not None:
             return serializer
-        for super_type in obj_type.__subclasses__():
+
+        for super_type in obj_type.__mro__:
```

Comment:
Can you add tests for this fix? You can add them to `CustomSerializationTestCase`

Reply:
Okay. Working on it.